src/components/ModelingMachineProvider.tsx

```diff
@@ -243,6 +244,17 @@ export const ModelingMachineProvider = ({
             return {}
           },
         }),
+        'Center camera on selection': () => {
+          engineCommandManager
+            .sendSceneCommand({
+              type: 'modeling_cmd_req',
+              cmd_id: uuidv4(),
+              cmd: {
+                type: 'default_camera_center_to_selection',
+              },
+            })
+            .catch(reportRejection)
```

Comment:
IMO should be in the CameraControls file but we can refactor later!

Reply:
Oh yeah good idea, that's better